README.md

```diff
@@ -138,6 +138,14 @@ W4G128 Average Accuracy of 13 tasks and Time Cost Results(Testing was conducted
 
 
 
+W2G64 Average Accuracy of 13 tasks and Time Cost Results(Testing was conducted on the Nvidia A100 80G using the version of PyTorch 2.6.0 with enable_torch_compile):
+
+| Model   | Qwen2.5-0.5B-Instruct | Falcon3-3B           | Qwen2.5-7B-Instruct  | Falcon3-10B          | Qwen2.5-72B-Instruct  |
+  |---------|-----------------------|----------------------|---------------------|----------------------|-----------------------|
```

Comment:
wrap it with `detail`, and better add detailed data for each task in doc folder later